cmd/kubelet/app/server.go

```diff
@@ -1216,10 +1216,13 @@ func RunKubelet(kubeServer *options.KubeletServer, kubeDeps *kubelet.Dependencie
 	// Setup event recorder if required.
 	makeEventRecorder(context.TODO(), kubeDeps, nodeName)
 
-	nodeIPs, err := nodeutil.ParseNodeIPArgument(kubeServer.NodeIP, kubeServer.CloudProvider)
+	nodeIPs, invalidNodeIps, err := nodeutil.ParseNodeIPArgument(kubeServer.NodeIP, kubeServer.CloudProvider)
```

Comment:
@bells17: I just spotted another `context.TODO` in line 1217 that can be replaced with the new `ctx` parameters.

I did another pass through all functions which now have such a parameter. I think no other `context.TODO` is left.

Reply:
/lgtm cancel

I'll re-add once this is done.

Reply:
@pohly Thanks for the catch! I've updated line 1217 with the new `ctx` parameter. Could you check again and give LGTM if it's all good?

Reply:
/lgtm

For the future: please avoid rebasing when squashing commits. https://github.com/kubernetes/kubernetes/compare/f301109a29e0de4b5f9988f0a49e0057be804a81..efdf04f8e5d0815862a24b6f7e1fb1c3be1a0173 now shows several unrelated changes, not just that one line that you changed.

Reply:
Now you need to rebase (merge conflict)...